### DISCUSSION TITLE
Template

### DISCUSSION THREAD

**Discussion by AishaKeyf** [dugsiiyeinc/dhaqsoPay#5] [Ideas]
Asc, dhallinyaro template kaan baa soo helay. hal iyo labo ka dhaha.

![WhatsApp Image 2024-11-13 at 23 34 42_8d0b5218](https://github.com/user-attachments/assets/b2ff222d-0f7d-4c6f-beab-c916b6d57275)

https://pin.it/4l2aokbBs

**Comment by abdinasir-Tman** [comment]
Engineer great waaye midkaan 
Options ka aan badsano si the best one aan u helno  inshaa allah

**Reply by abdinasir-Tman** [reply]
@dugsiiyeinc/dhaqsopay-contributors

**Reply by AishaKeyf** [reply]
Ok

**Comment by sharafdin** [comment]
eng waa hal design oo sawir uun ah. pages kale sidee loo qiimeyn ?
